### ISSUE TITLE
Look for configuration in hosting section of firebase.json

### ISSUE THREAD

**Issue by oscar-b** [firebase/superstatic#222]
The parameters for Superstatic when deploying to Firebase Hosting should be in the firebase.json file in a section called `hosting`:
https://firebase.google.com/docs/hosting/url-redirects-rewrites

It would be nice if Superstatic would use the same configuration from firebase.json when running locally.

**Comment by pwenzel** [comment]
@oscar-b 

I ran in to this issue as well. 

In the meantime, here is a workaround in the I used in my local Gulp+Browsersync setup. In my `gulpfile.js`, I manually loaded `firebase.json` and then merged its `hosting` section with the Superstatic config.

Snippet:

```javascript
var superstatic = require('superstatic');
var firebase_config = require('./firebase.json');

var superstatic_config = {
    stack: 'strict',
    config: firebase_config.hosting,
    cwd: process.cwd()
};

gulp.task('browser-sync', function() {
    browserSync.init({
        open: false,
        notify: true,
        ui: false,
        server: {
            baseDir: "./",
        },
        middleware: [superstatic(superstatic_config)]
    });
});
```

**Comment by oscar-b** [comment]
Hey

Forgot to update this, I ended up doing something like this (in `prodserver.js` for instance):

```javascript
const webpack = require('webpack');
const superstatic = require('superstatic').server;
const firebaseConfig = require('./firebase.json');

const serverConfig = {};
Object.assign(serverConfig, { config: firebaseConfig.hosting }, { port: 8080, host: '0.0.0.0' });
const app = superstatic(serverConfig);

app.listen(() => {
	console.log(`Listening on ${serverConfig.host}:${serverConfig.port}`);
});
```

But then I remembered that there's also `firebase serve` which also works perfectly :)

**Comment by mbleigh** [comment]
This was just fixed in #226
